spec/lib/mail_handler/mail_handler_spec.rb

```diff
@@ -336,6 +336,12 @@ def expect_header_string(fixture_file, header, header_string)
 
 describe 'when getting attachment attributes' do
 
+  it 'should handle a UTF-7 mail' do
+    mail = get_fixture_mail('utf-7-conversion.email')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.